src/util/string.h

```diff
@@ -251,11 +252,11 @@ inline std::string trim(const std::string &str)
 {
 	size_t front = 0;
 
-	while (std::isspace(str[front]))
+	while (std::iswspace(str[front]))
```

Comment:
I don't see how this should be useful, as we get an `std::string`, and the function takes a wide character? Same below.